pages/gallery.html

```diff
@@ -1,12 +1,150 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Taco Gallery</title>
+    <link rel="stylesheet" href="../styles/style.css" >
+    <link rel="stylesheet" href="../styles/gallery.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&ampdisplay=swap" rel="stylesheet"> 
 </head>
 <body>
-  
+    <!-- <div class="sticky-header">
+        <header>
+            <h1>Taco Gallery</h1>
+        </header>
+        <nav>
+            <ul>
+                <li>
+                    <a href="index.html">Home Page</a>
+                </li>
+                <li>
+                    <a href ="facts.html">Taco Facts</a>
+                </li>
+            </ul>
+        </nav>
+    </div> -->
```

Comment:
In industry, when you open a PR your code should be in a state as if it were ready to be merged into production code. Therefore we want to get rid of unused comments to keep the file from getting cluttered.